src/components/admin/Button/Button.tsx

```diff
@@ -22,12 +23,13 @@ export const Button: React.FC<ButtonProps> = ({
   loading = false,
   forwardRef,
   type = "button",
+  className,
   ...extraParams
 }) => (
   <button
     {...extraParams}
     type={type}
-    className={cn("Button", TW.general, TW[variant], TW[borders])}
+    className={cn("Button", TW.general, TW[variant], TW[borders], className)}
```

Comment:
Please see the comment at the place `className=""` is being used for alternative.

```suggestion
    className={cn("Button", TW.general, TW[variant], TW[borders])}
```

Reply:
[3469e91](https://github.com/sparkletown/sparkle/pull/2944/commits/3469e91693f840fa4d04232bc8c99b452b8f6b6d)